### DISCUSSION TITLE
Multiple `input.conf`  in Separated files !

### DISCUSSION THREAD

**Discussion by OmiChrome** [hooke007/mpv_PlayKit#285] [z_ENG_discuss]
So i used 
```
include ="~~/cache_mpv.conf"
include = "~~/conf-w\yt-mpv-network"
```
in mpv to break down my `mpv.conf` in multiple parts ,  so specific parts of conf is separate in different files 

but i was *looking* for same way at `INPUT.CONF` 

i tried  `input-conf = "~~/input_uosc-menu.conf" ` in many ways , but it din't work well , the issue was , 
mpv wasn't supporting `input.conf` and  `input_uosc-menu.conf` at the same time , since uosc takes so many options in input.conf i thought it would be better if it had seprated conf for this , 
as guessed i have **[blocked](https://mpv.io/manual/master/#options-no-input-default-bindings)** default binds , since i dont like them 

final say , 
i want `input.conf` in **broken down / Multiple conf** 
like this 
input_Equallizer.conf   # having equalizer keybinds as hue,contrast, etc 
input_seek.conf            # conf for seeking based keybinds 
many more in mind

**Comment by hooke007** [comment]
impossible

**Reply by OmiChrome** [reply]
💀

**Reply by OmiChrome** [reply]
i tried so many ways, even asked chatgpt < this failed , provided so many wrong options 

but :( 
man this is so sad

**Reply by OmiChrome** [reply]
lets make issue on mpv github :( after all that is what we can do !

**Reply by hooke007** [reply]
Asking chatgpt is wasting time. It is not designed to know how a program should work. If mpv allows you to use multi-inputconfs, then you could do that.

**Comment by OmiChrome** [comment]
here is the issue on mpv 
https://github.com/mpv-player/mpv/issues/11429

**Comment by OmiChrome** [comment]
just waiting for mpv https://github.com/mpv-player/mpv/issues/11429 response
